### DISCUSSION TITLE
Welcome to tiny-web-metaverse Discussions!

### DISCUSSION THREAD

**Discussion by takahirox** [takahirox/tiny-web-metaverse#76] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GabOnezio** [comment]
Hi, I'm recently new to Rust, I've only been using Rust for 3 years, I've never posted anything public until now with Rust, as I felt I wasn't prepared, but visualizing and understanding the power of the OpenSource community, I decided to open up here with the community that helped me interest, community that is interested in bringing people's reality to the Metaverse or something like that! Like 3D virtual world! and of course your profile fits right into this!
We will get along well even though I'm still young, pretend to learn and, if possible, teach!
